src/iuse.cpp

```diff
@@ -1841,6 +1841,32 @@ int iuse::purify_iv(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
+void spawn_spores( const player &p ) {
+    int spores_spawned = 0;
+    for ( const tripoint &dest : closest_tripoints_first( 4, p.pos() ) ) {
+        if ( g->m.move_cost( dest ) == 0 ) { // impassable terrain
+            continue;
+        }
+        float dist = trig_dist( dest, p.pos() );
+        if ( x_in_y( 1, dist ) ) { // slightly less frequent than previous code
```

Comment:
old code was effectively `!x_in_y(dist*dist,121)` which was not nearly enough falloff over the range of 4-6.